apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/ManualEvaluationResults/DocumentLogInfo/index.tsx

```diff
@@ -16,9 +16,8 @@ export function DocumentLogInfoForManualEvaluation({
   evaluation: EvaluationDto
   documentLog: DocumentLogWithMetadataAndErrorAndEvaluationResult
 }) {
-  // TODO: Somehow prevent the document log info from having the evaluations tab
   return (
-    <DocumentLogInfo documentLog={documentLog}>
+    <DocumentLogInfo documentLog={documentLog} hideEvaluations>
```

Comment:
Already commented in other place. We should not put data fetching concerns inside `DocumentLogInfo`. Pass tabs + tabs data to it. This component is only responsible of rendering the data and the visible tabs. Not fetching it. 

The one using it is reposonsible for that. 

For example in logs you fetch eval results and pass to it. 

But in evaluations you don't want that so you don't pass it